proposals/function-pointers.md

```diff
@@ -160,46 +192,104 @@ exactly what signature they are emitted with.
 ### Better function member
 The better function member specification will be changed to include the following line:
 
-> A `funcptr` is more specific than `void*`
+> A `func*` is more specific than `void*`
+
+This means that it is possible to overload on `void*` and a `func*` and still sensibly use the address-of operator.
+
+## Open Issues
+
+### NativeCallback Attribute
+This is an attribute used by the CLR to avoid the managed to native prologue when invoking. Methods marked by this 
```

Comment:
> This is an attribute used by the CLR to avoid the managed to native prologue when invoking.

The attribute add native to managed prolog to the method. It is not used to avoid anything.